app/main.py

```diff
@@ -23,6 +23,11 @@
     sentry_sdk.init(dsn=config.settings.sentry_dsn, traces_sample_rate=0.01)
     app.add_middleware(SentryAsgiMiddleware)
 
+if config.settings.enable_login_support:
+    logins.register_to_app(app)
```

Comment:
I'm confused as to why this is in your patch.  Did some rebasing add it?

Reply:
yeah